recipes/hail/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hail" %}
+{% set version = "0.2.34" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 9c3a4e5f81ad23b569c1d6dc80bfb070
+  
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Is a C-compiler needed to build this package? I don't see any C-extensions in the `setup.py`